assets/js/src/sdk/components/index.ts

```diff
@@ -153,6 +153,10 @@ export * from '@Pimcore/components/toolstrip/box/tool-strip-box'
 export * from '@Pimcore/components/tree-element/tree-element'
 export * from '@Pimcore/components/users-roles-dropdown/users-roles-dropdown'
 export * from '@Pimcore/components/vertical-timeline/vertical-timeline'
+export type { TabpanelItem } from '@Pimcore/components/tabpanel/tabpanel'
+
+// @todo check if this component should really be branded to workflows or if we can find a mor re-usable name
```

Comment:
We removed the workflow-card on purpose from the SDK, please just remove it again for now. If at some point we create a generic component out of it, we can re-add it.

Reply:
ah oke in my version it was still there - but good to know :) i removed it.

https://github.com/pimcore/studio-ui-bundle/pull/2411/commits/d49967f80033e8e42d1b1ff5e9701750a1b91782